ci-operator/step-registry/ipi/conf/gcp/oidc-creds-deprovision/ipi-conf-gcp-oidc-creds-deprovision-commands.sh

```
@@ -27,7 +27,13 @@ echo "RELEASE_IMAGE_LATEST_FROM_BUILD_FARM: ${RELEASE_IMAGE_LATEST_FROM_BUILD_FA
 # for deprovision.
 # https://docs.ci.openshift.org/docs/architecture/step-registry/#sharing-data-between-steps
 echo "> Extract gcp credentials requests from the release image"
-oc adm release extract --credentials-requests --cloud=gcp --to="/tmp/credrequests" "${RELEASE_IMAGE_LATEST_FROM_BUILD_FARM}"
+dir=$(mktemp -d)
+pushd "${dir}"
+cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)